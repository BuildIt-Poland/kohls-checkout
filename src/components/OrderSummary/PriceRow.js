import styled from 'styled-components';

const PriceRow = styled.div`
  width: 100%;
  display: flex;
  justify-content: space-between;
`;

export default PriceRow;
