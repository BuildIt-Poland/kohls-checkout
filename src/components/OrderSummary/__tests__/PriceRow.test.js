import React from 'react';
import { create } from 'react-test-renderer';

import PriceRow from '../PriceRow';

describe('COMPONENT - OrderSummary PriceRow', () => {
  it('render NextStep component', () => {
    const component = create(<PriceRow />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
