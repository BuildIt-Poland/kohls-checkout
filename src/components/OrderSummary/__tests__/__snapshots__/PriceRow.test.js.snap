// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - OrderSummary PriceRow render NextStep component 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;
